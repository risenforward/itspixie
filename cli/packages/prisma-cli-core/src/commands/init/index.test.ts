import * as nock from 'nock'
import { Config } from 'prisma-cli-engine'
import Init from './'

afterAll(() => {
  nock.cleanAll()
})

describe.skip('init', () => {
  test('test project', async () => {
    const result = await Init
      .mock
      //'-t',
      //'blank',
      ()
    expect(result.out.stdout.output).toContain('Written files:')
  })
})
