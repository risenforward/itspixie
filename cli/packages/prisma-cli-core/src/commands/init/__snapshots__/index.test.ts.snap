// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init choose local 1`] = `
Object {
  "datamodel.graphql": "type User {
  id: ID! @unique
  name: String!
}
",
  "docker-compose.yml": "version: '3'
services:
  prisma:
    image: prismagraphql/prisma:experimental
    restart: always
    ports:
    - \\"4466:4466\\"
    environment:
      CLUSTER_ADDRESS: \\"\\"
      SCHEMA_MANAGER_SECRET: \\"\\"
      SCHEMA_MANAGER_ENDPOINT: \\"\\"
      BUGSNAG_API_KEY: \\"\\"
      PRISMA_CONFIG: |
        port: 4466
        databases:
          default:
            connector: mysql
            active: true
            host: db
            port: 3306 
            user: root
            password: prisma
  db:
    image: mysql:5.7
    restart: always
    environment:
      MYSQL_USER: prisma
      MYSQL_ROOT_PASSWORD: prisma
",
  "prisma.yml": "endpoint: http://localhost:4466
datamodel: datamodel.graphql",
}
`;

exports[`init choose local 2`] = `
"Created 3 new files:                                                                          

  prisma.yml           Prisma service definition
  datamodel.graphql    GraphQL SDL-based datamodel (foundation for database)
  docker-compose.yml   Docker configuration file

Run docker-compose up -d. Then you can run $ prisma deploy to deploy your database service.
"
`;

exports[`init test project 1`] = `Object {}`;

exports[`init test project 2`] = `""`;
